source/includes/_fonenumbers.md

```diff
@@ -18,41 +18,29 @@ $ curl --request GET
 ```json
 {
   "statusCode": 200,
-  "Fonenumbers": [
+  "fonenumbers": [
     {
-      "Number": [
-        "3094399908"
-      ],
-      "RegionBlock": [
-        "na"
-      ],
-      "CNAMLookup": [
-        "yes"
-      ],
-      "AccountCode": [
-        "2005551524"
-      ],
-      "ServiceOption": [
-        "475744919148"
-      ],
-      "ServiceType": [
-        "Device"
-      ],
-      "T38": [
-        "yes"
-      ],
-      "SMS": [
-        "no"
-      ],
-      "PortedNumber": [
-        "no"
-      ],
-      "ActivateDate": [
-        "2017-01-06 10:09:51"
-      ],
-      "RewriteTo": [
-        ""
-      ]
+      "fonenumber": "8889807630",
+      "serviceType": "TimeOfDayRoute",
+      "receive": {
+        "url": "https://hookb.in/Ew7WzAb2",
+        "method": "JSON",
+        "username": "johndoe",
+        "password": "secret"
```

Comment:
This return data is completely different from the previous return data, I was wondering why I hadn't noticed that before.